lib/sequences/07_argonaut_search_sequence.rb

```diff
@@ -166,7 +167,13 @@ def validate_reply(klass, reply)
           'http://www.fhir.org/guides/argonaut/r2/Conformance-server.html',
           'All servers SHALL make available the read interactions for the Argonaut Profiles the server chooses to support.' do
 
-    todo
+    assert !@allergyintolerance.nil?, 'Expected valid DSTU2 AllergyIntolerance resource to be present'
+    id = @allergyintolerance.try(:id)
```

Comment:
Technically `.try()` is not necessary here, because we know at this point that `@allergyintolerance` cannot be `nil`.  I think this is a minor enough point that we can leave this though.  Just something to keep in mind in the future.